ios/FluentUI/Vnext/List/ListCell.swift

```diff
@@ -27,19 +27,20 @@ import SwiftUI
     @objc @Published public var leadingView: UIView?
     @objc @Published public var leadingViewSize: MSFListCellLeadingViewSize = .medium
     @objc @Published public var title: String = ""
-    @objc @Published public var subtitle: String?
+    @objc @Published public var subtitle: String = ""
     @objc @Published public var trailingView: UIView?
     @objc @Published public var accessoryType: MSFListAccessoryType = .none
     @objc @Published public var titleLineLimit: Int = 0
     @objc @Published public var subtitleLineLimit: Int = 0
     @objc @Published public var children: [MSFListCellState]?
     @objc @Published public var isExpanded: Bool = false
-    @objc @Published public var layoutType: MSFListCellLayoutType = .oneLine
+    @objc @Published public var layoutType: MSFListCellLayoutType = .none
     @objc public var onTapAction: (() -> Void)?
 }
 
 /// Pre-defined layout heights of cells
 @objc public enum MSFListCellLayoutType: Int, CaseIterable {
+    case none
```

Comment:
It seems like the implementation you've done for this is more like "automatic" than "none" because it considers the subtitle string to define the height.

Thoughts?